compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/selection/SelectionHandles.kt

```diff
@@ -84,21 +84,6 @@ internal expect fun SelectionHandle(
     modifier: Modifier,
 )
 
-@Composable
-internal fun SelectionHandle(
```

Comment:
Why is it removed?

Reply:
I see. To fail on compilation state, if it is added somewhere without us noticing